suites/pacific/cephadm/tier-1_cephadm_cluster_view.yaml

```diff
@@ -0,0 +1,87 @@
+#===============================================================================================
+#--------------------------------------
+#----- Ceph code changes in 5.1 -------
+#--------------------------------------
+#--------------------------------------
+# Conf: conf/pacific/cephadm/sanity-cephadm.yaml
+#
+# Tier-1 test cases for
+#    - Bootstrap a cluster
+#    - Add OSD nodes and MON nodes
+#    - Verify View of Cluster and MON IPs updated in ceph.conf
+#===============================================================================================
+
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
```

Comment:
Don't need multiple tests for configuring cluster. Refer config from PR - https://github.com/red-hat-storage/cephci/pull/2285/files#diff-30d0c71638719cd21ab61374f7513b8eab53158fd0960189b1795cc95400099dR20

Reply:
Okay, I'll consider this!

Reply:
Made the changes as requested.